content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +251,47 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Scheduler configuration migrations
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1beta1 → v1beta2" %}}
+* With the v1beta2 configuration version, you can use a new score extension for the
+  `NodeResourcesFit` plugin.
+  The new extension combines the functionalities of the `NodeResourcesLeastAllocated`,
+  `NodeResourcesMostAllocated` and `RequestedToCapacityRatio` plugins.
+  For example, if you previously used the `NodeResourcesMostAllocated` plugin, you
+  would instead use `NodeResourcesFit` (enabled by default) and add a `pluginConfig`
+  with a `scoreStrategy` that is similar to:
+  ```yaml
+  apiVersion: kubescheduler.config.k8s.io/v1beta2
+  kind: KubeSchedulerConfiguration
+  profiles:
+  - pluginConfig:
+    - args:
+        scoringStrategy:
+          resources:
+          - name: cpu
+            weight: 1
+          type: MostAllocated
+      name: NodeResourcesFit
+  ```
+
+* The scheduler plugin `NodeLabel` is deprecated; instead, use the [`NodeAffinity`](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) plugin (enabled by default) to achieve similar behavior.
+
+* The scheduler plugin `ServiceAffinity` is deprecated; instead, use the [`InterPodAffinity`](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity) plugin (enabled by default) to achieve similar behavior.
+
+* The scheduler plugin `NodePreferAvoidPods` is deprecated; instead, use [node taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) to achieve similar behavior.
+
+* A plugin enabled in a v1beta2 configuration file takes precedence over the default configuration for that plugin. This simplifies enabling default plugins with custom configuration without the necessary to explicitly disable those default plugins.
```

Comment:
nit: Could you clarify the second sentence on line 285 or possibly remove that sentence?
This simplifies enabling default plugins with custom configurations without explicitly disabling those default plugins?

Reply:
Rmoved.